maro/cli/data_pipeline/citi_bike.py

```diff
@@ -27,6 +27,23 @@
 
 
 class CitiBikePipeline(DataPipeline):
+    """Generate citi_bike data bin and other necessary files for the specified topology from specified source.
+    They will be generated in ~/.maro/data/citi_bike/[topology]/_build.
```

Comment:
Same to the comments above: blank line below the summary line; start each arg with capital letter....

Same to other docstring, please check.